src/Servers/IIS/IIS/test/Common.FunctionalTests/Inprocess/StartupTests.cs

```diff
@@ -44,8 +43,6 @@ public async Task ExpandEnvironmentVariableInWebConfig()
 
         [ConditionalTheory]
         [InlineData("bogus", "", @"Executable was not found at '.*?\\bogus.exe")]
-        [InlineData("c:\\random files\\dotnet.exe", "something.dll", @"Could not find dotnet.exe at '.*?\\dotnet.exe'")]
-        [InlineData(".\\dotnet.exe", "something.dll", @"Could not find dotnet.exe at '.*?\\.\\dotnet.exe'")]
```

Comment:
These no longer valid?

Reply:
I was lazy here I think. The input is invalid and should be tested but the error message won't be the same.